pydgin/elf.py

```diff
@@ -528,28 +528,23 @@ def elf_reader( file_obj, is_64bit=False ):
     #section_name = start.partition('\0')[0]
     section_name = start.split('\0', 1)[0]
 
-    # only sections marked as lloc should be written to memory
-
-    if not (shdr.flags & ElfSectionHeader.FLAGS_ALLOC):
-      continue
-
     # Read the section data if it exists
-
-    if section_name not in ['.sbss', '.bss']:
+    if (shdr.type == ElfSectionHeader.TYPE_NULL or
+        section_name == '.comment' or
```

Comment:
Hmm, I'm a bit worried this might break other ISAs since they sometimes have other sections which shouldn't be loaded and can corrupt other sections if loaded. Unfortunately Travis might not catch these since assembly tests we use in Travis might not have those ELF sections. I'm not an expert on ELF format but maybe this is a bug in their cross-compiler and ISS? Is this section supposed to be executable or just data?